cv-data-service-library/src/main/java/com/trihydro/library/helpers/JsonToJavaConverter.java

```diff
@@ -390,10 +400,10 @@ public OdeTravelerInformationMessage.DataFrame.Region.Path GetPathData(JsonNode
             if (nodesNode == null)
                 return null;
 
-            JsonNode nodeXYArrNode = isXy ? nodesNode.get("NodeXY") : nodesNode.get("NodeLL");
+            JsonNode nodeXYArrNode = isXy ? nodesNode.get("NodeXY") : nodesNode;
```

Comment:
Question (blocking): Are NodeLL nodes supported also?

Reply:
Yes, they are. There is no longer a nested "NodeLL" object under the "nodes" array, which is why this was modified.